home/pi/setup.sh

```diff
@@ -79,6 +79,12 @@ sudo apt-get install -y i2c-tools
 sudo raspi-config nonint do_i2c 0
 
 # LED Support
+source /home/pi/mycroft-core/.venv/bin/activate
+git clone --depth 1 https://github.com/respeaker/pixel_ring.git
+cd pixel_ring
+python setup.py install
+cd ~
+
```

Comment:
maybe use `pip install git+https://github.com/respeaker/pixel_ring.git` to install it?

Reply:
Great suggestion! Adding to mark-2-pi requirements.txt and closing this PR